core/aws_ddk_core/stages/glue_transform.py

```diff
@@ -123,6 +124,11 @@ def __init__(
                 **glue_crawler_args,
             )
             crawler_name = self._crawler.ref
+        crawler_arn = f"""
+            arn:{cdk.Stack.of(self).partition}:
+            glue:{cdk.Stack.of(self).region}:{cdk.Stack.of(self).account}:
```

Comment:
I think just `self.region` should work

Reply:
When I tried that I got: `AttributeError: 'GlueTransformStage' object has no attribute 'region'`

Reply:
Ahh right, sorry, it's `scope.region` - the scope is the stack while self here refers to the stage itself

Reply:
There's no guarantee that `scope` will be a Stack. This stage could be constructed from within an L3 construct. The `cdk.Stack.of` function will go up the construct tree until it reaches the stack.

In order to make this code cleaner however, you could just extract stack as a separate variable, thus removing the need to invoke `cdk.Stack.of` each time.